pages/services/kubernetes/1.0.0-1.9.3/cluster_sizing/index.md

```diff
@@ -0,0 +1,185 @@
+---
+layout: layout.pug
+navigationTitle: Cluster Sizing
+title: Cluster Sizing
+menuWeight: 45
+excerpt:
+---
+
+<!-- This source repo for this topic is https://github.com/mesosphere/dcos-kubernetes -->
+
+
+The Mesosphere Kubernetes Service supports clusters that meet all the following
+criteria:
+
+* No more than 100 nodes
+* No more than 10000 total pods
+* No more than 20000 total containers
+* No more than 100 pods per node
+* No more than 10 pods per core running on a node
+
+Note that these values are based on lightweight pods or containers like nginx.
+It is up to the cluster operator to size the Kubernetes cluster appropriately
+for their specific workloads.
+
+## Planning
+
+Before deploying your Kubernetes cluster, it is important to consider what size
+cluster you require. It is the responsibilty of the DC/OS cluster to ensure
```

Comment:
It is the responsibility of the DC/OS cluster operator to ensure there are enough resources available in the DC/OS cluster to satisfy the Kubernetes cluster deployment.

Reply:
Done.